docs/writing_a_plugin/import_format.md

```diff
@@ -43,24 +43,24 @@ CharA, 2.4
 CharB, 1.6
 ```
 
-A *SimpleTxt* file always represents a single measurement. The format starts with `#Header` as first line in the file. The following lines contain general information about the measurement (for example the date of the measurement or the operator of the measuring machine) in the form of name value pairs separated by a colon. The value part of the file begins with the line `#Characterstic,Value`. Each following line consists of a characteristic name and its coresponding measured value separated by a comma. A *SimpleTxt* file can contain as many general information entries and measured characteristics as necessary.
+A *SimpleTxt* file always represents a single measurement. Files of this format always start with `#Header` as first line. Then the following lines contain general information about the measurement in the form of name value pairs separated by a colon. For example the date of the measurement or some descriptive text. The value part of the file begins with the line `#Characterstic,Value`. Each following line consists of a characteristic name and its coresponding measured value separated by a comma. A *SimpleTxt* file can contain as many general information entries and measured characteristics as necessary.
```

Comment:
* Remove "always" from the second sentence: Files of this format ~always~ start with... 
* Write "corresponding" with two 'r'

Reply:
Done.